frontend/components.js

```diff
@@ -628,6 +630,22 @@ class SendAdaClass extends Component {
             {class: 'validationMsg send-amount-validation-error'},
             translations[sendAmountValidationError.code](sendAmountValidationError.params)
           ),
+        allFundsSendable
+          ? h(
```

Comment:
too much repetition, this could be merged into one element and use the ternary operator only on the second parameter, something like this:
`
h('a', allFundsSendable ? {class: 'intro-link ...', onClick: '...'} : {class: 'disabled-link'}, 'send all funds')`